src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v10\v0;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Core\CacheManager;
+
+/**
+ * @see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Deprecation-88366-DefaultCachingFrameworkCacheNamesChanged.html
+ */
+class ChangeDefaultCachingFrameworkNamesRector extends AbstractRector
+{
+    /*
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, CacheManager::class)) {
```

Comment:
This check isn't working like I expected, always hit this one, If I remove this check and the 
```php 
if (! $this->isName($node->name, 'getCache')) {
    return null;
}
``` 

Then I can see that the change of the argument is working as expected. So not that far from something working. But there might be a simpler rule for this one too as mentioned in the comment on the PR itself.

Reply:
@tomasnorre The things is this line here use TYPO3\CMS\Core\Core\CacheManager; Remove one of the core subprefix and it should work